<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherHome</class>
 <widget class="QMainWindow" name="TeacherHome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Main window background */
QMainWindow {
    background-color: #f0f0f0; /* Светло-серый фон */
}

/* General button style */
QPushButton {
    background-color: #4CAF50; /* Зеленый фон */
    border: none; /* Без границы */
    color: white; /* Белый текст */
    padding: 10px 24px; /* Внутренний отступ */
    text-align: center; /* Выравнивание текста по центру */
    font-size: 16px; /* Размер шрифта */
    margin: 4px 2px; /* Внешний отступ */
    border-radius: 12px; /* Закругленные углы */
}

QPushButton:hover {
    background-color: #45a049; /* Темно-зеленый фон при наведении */
}

QPushButton:pressed {
    background-color: #398439; /* Ещё темнее при нажатии */
}

/* Line edit fields */
QLineEdit {
    border: 1px solid #ccc; /* Светло-серая граница */
    border-radius: 10px; /* Закругленные углы */
    padding: 10px; /* Внутренний отступ */
    font-size: 16px; /* Размер шрифта */
    background-color: #fff; /* Белый фон */
    margin: 4px 0; /* Внешний отступ */
}

/* Line edit focus style */
QLineEdit:focus {
    border: 1px solid #4CAF50; /* Зеленая граница при фокусе */
}

/* Labels */
QLabel {
    font-size: 16px; /* Размер шрифта */
    color: #333; /* Темно-серый текст */
    margin: 4px 0; /* Внешний отступ */
}

/* Combobox */
QComboBox {
    border: 1px solid #ccc; /* Светло-серая граница */
    border-radius: 10px; /* Закругленные углы */
    padding: 8px; /* Внутренний отступ */
    font-size: 16px; /* Размер шрифта */
    background-color: #fff; /* Белый фон */
    margin: 4px 0; /* Внешний отступ */
}

/* Combobox focus style */
QComboBox:focus {
    border: 1px solid #4CAF50; /* Зеленая граница при фокусе */
}

/* Table headers */
QHeaderView::section {
    background-color: #4CAF50; /* Зеленый фон */
    color: white; /* Белый текст */
    padding: 4px; /* Внутренний отступ */
}

/* Table items */
QTableWidget {
    border: 1px solid #ccc; /* Светло-серая граница */
    border-radius: 10px; /* Закругленные углы */
    font-size: 14px; /* Размер шрифта */
    background-color: #fff; /* Белый фон */
    margin: 4px 0; /* Внешний отступ */
}

/* Table item selection */
QTableWidget::item:selected {
    background-color: #45a049; /* Темно-зеленый фон при выборе */
}

/* Message box buttons */
QMessageBox QPushButton {
    background-color: #4CAF50; /* Зеленый фон */
    color: white; /* Белый текст */
    border: none; /* Без границы */
    padding: 10px 20px; /* Внутренний отступ */
    border-radius: 5px; /* Закругленные углы */
}

QMessageBox QPushButton:hover {
    background-color: #45a049; /* Темно-зеленый фон при наведении */
}

QMessageBox QPushButton:pressed {
    background-color: #398439; /* Ещё темнее при нажатии */
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Журнал</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Поставить оценку:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Студент:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="GradeEdit">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>150</y>
        <width>186</width>
        <height>53</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>186</width>
        <height>53</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>110</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Оценка:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>30</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Факультатив:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxGeade">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>60</y>
        <width>186</width>
        <height>53</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>186</width>
        <height>53</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushGrade">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>270</y>
        <width>221</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Поставить оценку</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>271</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>74</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Имя</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="NameStudentEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>74</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Фамилия</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="SurnameStudentEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Отчество</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="PatronumicStudentEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Просмотреть статистику</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>281</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="minimumSize">
            <size>
             <width>74</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Имя</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="NameStudentViewEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="minimumSize">
            <size>
             <width>74</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Фамилия</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="SurnameStudentViewEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Отчество</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="PatronumicStudentViewEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ВВедите данные студента:</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="RaspisanietableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>381</width>
        <height>121</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Название</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Вид проведения</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Время</string>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="GradetableWidget">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>240</y>
        <width>291</width>
        <height>121</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Название</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Оценка</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="ViewButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>90</y>
        <width>381</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Посмотреть расписание и успеваемость</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Профиль</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Профиль:</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>40</y>
        <width>501</width>
        <height>192</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Название</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Вид проведения</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Время</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Группа</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Факультативы:</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>261</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="minimumSize">
            <size>
             <width>74</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Имя</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="NamePrEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="minimumSize">
            <size>
             <width>74</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Фамилия</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="SurnamePrEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Отчество</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="PatronumicPrEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Запись</string>
     </attribute>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Заполните данные о студенте:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ZapispushButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>120</y>
        <width>271</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Записать на факультатив</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>301</width>
        <height>245</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_17">
             <property name="minimumSize">
              <size>
               <width>74</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Имя</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="NameStudentZapisEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_18">
             <property name="minimumSize">
              <size>
               <width>74</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Фамилия</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="SurnameStudentZapisEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="label_19">
             <property name="minimumSize">
              <size>
               <width>74</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Отчество</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="PatronumicStudentZapisEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="label_20">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>95</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Факультатив</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="ZapisStudentcomboBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>53</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>801</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Выйти из сессии</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
